packages/apps/src/configs/wallets/chain-config.ts

```diff
@@ -28,11 +28,11 @@ export const chainsConfig: AppConfig['chains'] = {
     logo: EdgwareLogo,
     currencies: [
       {
-        currencyId: 1,
+        currencyId: 2,
```

Comment:
CurrencyIDs on Substrate are identifiers for non-native tokens. We reserve one for the native token to not interfere and otherwise choose higher values (though we'll see how other chains handle this in time).